src/subhalo.cpp

```diff
@@ -246,9 +246,6 @@ void Subhalo_t::AverageCoordinates()
     copyHBTxyz(ComovingMostBoundPosition, Particles[GetTracerIndex()].ComovingPosition);
     copyHBTxyz(PhysicalMostBoundVelocity, Particles[GetTracerIndex()].GetPhysicalVelocity());
 
-    /* Copy the mass, in case this object becomes an orphan in the current output. */
```

Comment:
This was making zero difference, since the a call to `CountParticles()` would reset the bound mass to what it had before.